domain/status/errors/errors.go

```diff
@@ -14,6 +14,14 @@ const (
 	// an application that is dead.
 	ApplicationIsDead = errors.ConstError("application is dead")
 
+	// RelationNotFound describes an error that occurs when the relation
+	// being operated on does not exist.
+	RelationNotFound = errors.ConstError("relation not found")
```

Comment:
I have been happily importing other domain error packages in the removal domain instead of creating duplicate error types across packages.

I will do an audit of this and see where we stand.